noir-projects/noir-contracts/contracts/app/simple_token_contract/src/main.nr

```diff
@@ -29,7 +29,6 @@ pub contract SimpleToken {
     };
 
     use dep::uint_note::uint_note::{PartialUintNote, UintNote};
-    use aztec::protocol_types::traits::ToField;
```

Comment:
Quite annoying that we have this contract to maintain. Had to do the updates here since the same note was used.